scripts/wfl/twisttcap/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM gcr.io/google.com/cloudsdktool/google-cloud-cli:404.0.0-slim
```

Comment:
For own knowledge: Is it possible to start from a different docker image say 
> FROM python:3.8-slim-buster
and install the `cloudsdk` through PIP or APT-GET? The potential would be a smaller image output